packages/chat-stateful-client/package.json

```diff
@@ -40,11 +40,11 @@
     "nanoid": "3.3.6"
   },
   "peerDependencies": {
-    "@azure/communication-chat": ">=1.4.0"
+    "@azure/communication-chat": "1.5.0-beta.1 || >=1.4.0"
   },
   "devDependencies": {
-    "@azure/communication-chat": "^1.4.0",
-    "@azure/communication-signaling": "1.0.0-beta.21 || 1.0.0-beta.20",
+    "@azure/communication-chat": "1.5.0-beta.1 || ^1.4.0",
+    "@azure/communication-signaling": "1.0.0-beta.23 || 1.0.0-beta.22",
```

Comment:
@JamesBurnside @azure/communication-signaling@1.0.0-beta.22 is a dependency change that affects this stable. This is needed to match @azure/communication-chat@1.4.0 that this current stable release is using.

Reply:
`@azure/communication-signaling` is only a devDepenency and we only use it for types in tests - so shouldn't be impacting the stable package.
But also I'm down for updating it - no harm in it!
But we should _not_ have these changes (so either manually reverting them or a separate PR, whichever is easier):
![image](https://github.com/Azure/communication-ui-library/assets/2684369/47c7ce68-9f3f-43ce-aca1-91f47481ad28)

cc @Leah-Xia-Microsoft

Reply:
Ah good point. It's only a dev dependency so it won't affect the npm package.

Reply:
Good point. Since separating this PR might cause unexpected issues, and since it's ok to have signaling 21 as just devDepenency, we will not cherry-pick this PR into the release branch.